roundabout/search/views.py

```diff
@@ -445,7 +444,6 @@ def get_avail_fields():
                         dict(value="location__name",            text="Location", legal_lookups='STR_LOOKUP'),
                         dict(value="assembly__description",  text="Description", legal_lookups='STR_LOOKUP'),
                         dict(value="build_notes",                  text="Notes", legal_lookups='STR_LOOKUP'),
-                        dict(value="detail",                      text="Detail", legal_lookups='STR_LOOKUP'),
```

Comment:
Keep this. Call it "Notes & Details" to keep consistent with searching on "detail" as we do in Inventory.

Reply:
Removed again due to functionality issues & desire to take better approach... see comment below.